{"name": "test_login_params[data_for_login4]", "status": "passed", "attachments": [{"name": "log", "source": "c63a50c2-b1e6-4923-a61a-a78f7bccfc27-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('', '', 'Fail')"}], "start": 1745923018355, "stop": 1745923020634, "uuid": "3a0f62cb-4022-4d8a-bb07-ee74a143f825", "historyId": "ff385ada907923126a299abd0a35750e", "testCaseId": "47f970e602e4eefb1a3e5455dd555a36", "fullName": "testCases.test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "14772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login_params"}]}